server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -101,22 +67,19 @@
             clusterAlias,
             transportService,
             connectionManager,
-            REMOTE_NODE_CONNECTIONS.get(connectionSettings),
             getNodePredicate(nodeSettings),
-            REMOTE_CLUSTER_SEEDS.get(connectionSettings)
+            RemoteCluster.REMOTE_CLUSTER_SEEDS.get(connectionSettings)
         );
     }
 
     SniffConnectionStrategy(String clusterAlias,
                             TransportService transportService,
                             RemoteConnectionManager connectionManager,
-                            int maxNumRemoteConnections,
```

Comment:
out of curiosity: whats the concrete reason to remove this and hardcode the number of connections?

Reply:
No particular strong reason. A mix of trying to normalize the settings between sniff and pg-tunneling, not being obvious why/when you'd want to change it and the documentation.